test/test_qos.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+
+from kappe.module.qos import (
+    DurabilityPolicy,
+    HistoryPolicy,
+    LivelinessPolicy,
+    Qos,
+    QosDuration,
+    ReliabilityPolicy,
+    dump_qos_list,
+    parse_qos_list,
+)
+
+
+def test_parse_qos_list_single():
+    """Test parsing a single QoS profile from YAML."""
+    yaml_str = """
+- history: 1
+  depth: 10
+  reliability: 2
+  durability: 2
+"""
```

Comment:
My favourite method from `textwrap`.
```suggestion
    yaml_str = dedent("""
        - history: 1
          depth: 10
          reliability: 2
          durability: 2
    """)
```